.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,30 @@
+name: "Pre commit"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
+    steps:
+     - uses: actions/checkout@v2
+
+     - name: Set up Python ${{ matrix.python-version }}
+       uses: actions/setup-python@v2
+       with:
+         python-version: ${{ matrix.python-version }}
+
+     - name: Install pre-commit
+       shell: bash
+       run: pip3 install pre-commit
```

Comment:
[Pre-commit](https://pre-commit.com/hooks.html) is a tool to apply certain conventions - not only for Python. You can install it locally with:

```bash
pip3 install pre-commit
pre-commit install
```

Then all checks are applied before you commit to keep them clean.